src/components/Label.tsx

```diff
@@ -9,10 +9,16 @@ interface ILabel extends IText<HTMLLabelElement> {
   variant?: Variant;
 }
 
-export const Label: React.FunctionComponent<ILabel> = ({ htmlFor, variant, children }) => {
+export const Label: React.FunctionComponent<ILabel> = ({ htmlFor, variant, children, disabled = false }) => {
   const fg = useFg(variant);
+  const disabledStyles = disabled
+    ? {
+        opacity: 0.6,
+        cursor: 'not-allowed',
+      }
+    : {};
```

Comment:
```suggestion
    : null;
```

Reply:
Will that work with the spread operator?
```js
{...null}
```
Whaaaat it works. Why does it work?

Reply:
Didn't check the spec, but object rest spread operators works with any (or almost) any value. Boolean would work too, for sure.